base-cert-manager/certificates.yaml

```diff
@@ -0,0 +1,61 @@
+# https://cert-manager.io/docs/configuration/selfsigned/#bootstrapping-ca-issuers
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: cockroachdb-selfsigned-issuer
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: cockroachdb-ca
+spec:
+  isCA: true
+  commonName: cockroachdb-ca
+  dnsNames:
+    - cockroachdb-ca
+  secretName: cockroachdb-ca
+  issuerRef:
+    name: cockroachdb-selfsigned-issuer
+    kind: Issuer
+    group: cert-manager.io
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: cockroachdb-ca-issuer
+spec:
+  ca:
+    secretName: cockroachdb-ca
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: node
+spec:
+  issuerRef:
+    name: cockroachdb-ca-issuer
+    kind: Issuer
+  commonName: node
+  dnsNames:
```

Comment:
The cfssl node certs include `localhost` and `127.0.0.1` - I don't know if anyone uses the sql client against localhost by exec'ing into the pod or port-forwarding but might as well add them in just in case